doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -399,6 +400,18 @@
     }
     ```
 
+20. Renamed `BaseRelativePointer` to `UntypedRelativePointer`
+
+    ```cpp
+    // before
+    #include "iceoryx_hoofs/internal/relocatable_pointer/base_relative_pointer.hpp"
+    iox::rp::BaseRelativePointer myUntypedRelativePointer;
+
+    // after
+    #include "iceoryx_hoofs/memory/relative_pointer.hpp" /// @todo #605 adapt namespace and directory
+    iox::memory::UntypedRelativePointer myUntypedRelativePointer;
```

Comment:
This `memory` namespace is not where the `UntypedRelativePointer` is uniquely defined though. Later on it is used from `rp` namespace at least.

Reply:
This will happen on a follow-up PR.